oxidation/libparsec/crates/tests_fixtures/src/lib.rs

```diff
@@ -12,275 +12,11 @@ pub use rstest;
-pub fn device_certificate(alice: &Device, bob: &Device, timestamp: DateTime) -> Vec<u8> {
-    DeviceCertificate {
-        author: CertificateSignerOwned::User(alice.device_id.clone()),
-        timestamp,
-        device_id: bob.device_id.clone(),
-        device_label: bob.device_label.clone(),
-        verify_key: bob.verify_key(),
-    }
-    .dump_and_sign(&alice.signing_key)
-}
-
-#[fixture]
-#[once]
-pub fn redacted_device_certificate(alice: &Device, bob: &Device, timestamp: DateTime) -> Vec<u8> {
-    DeviceCertificate {
-        author: CertificateSignerOwned::User(alice.device_id.clone()),
-        timestamp,
-        device_id: bob.device_id.clone(),
-        device_label: None,
-        verify_key: bob.verify_key(),
-    }
-    .dump_and_sign(&alice.signing_key)
-}
-
-/// A temporary path that will be removed on drop.
-pub struct TmpPath(PathBuf);
-
-impl std::ops::Deref for TmpPath {
-    type Target = PathBuf;
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
-}
-
-impl Drop for TmpPath {
-    fn drop(&mut self) {
-        if let Err(err) = std::fs::remove_dir_all(&self.0) {
-            // Cannot remove the directory :'(
-            // If we are on Windows, it most likely means a file in the directory
-            // is still opened. Typically a SQLite database is still opened because
-            // the SQLiteExecutor's drop doesn't wait
-            let content = {
-                match std::fs::read_dir(&self.0) {
-                    Ok(items) => items
-                        .into_iter()
-                        .map(|item| match item {
-                            Ok(item) => {
-                                format!("{}", item.path().strip_prefix(&self.0).unwrap().display())
-                            }
-                            Err(err) => format!("<error: {:?}>", err),
-                        })
-                        .collect(),
-                    Err(_) => vec!["<empty>".to_owned()],
-                }
-                .join(" ")
-            };
-            panic!(
-                "Cannot remove {:?}: {}\n\
-                Content: {}\n\
-                Have you done a gracious close of resources in your test ?",
-                &self.0, &err, content
-            );
-        }
-    }
-}
-
-#[fixture]
-pub fn tmp_path() -> TmpPath {
-    let mut path = std::env::temp_dir();
-
-    path.extend(["parsec-tests", &Uuid::new_v4().to_string()]);
-
-    std::fs::create_dir_all(&path).expect("Cannot create tmp_path dir");
-
-    TmpPath(path)
-}
-
-// Most unit tests uses the current time as a shorthand to get a datetime object.
-// This is something that is cumbersome (by design !) in our code given it is
-// achieved by doing `TimeProvider::default().now()`.
-// So instead this fixture should be used when a default `DateTime` object is needed.
-#[fixture]
-pub fn timestamp() -> DateTime {
-    "2020-01-01T00:00:00Z".parse().unwrap()
-}
+pub use libparsec_tests_utils::{tmp_path, TmpPath};
```

Comment:
Add comment
```rust
// Re-expose fixtures
```

Reply:
The `pub use` already provide that information

Reply:
> The comment is already present (above `pub use rstest`)